uk_bin_collection/uk_bin_collection/councils/BirminghamCityCouncil.py

```
@@ -1,28 +1,39 @@
 from typing import Dict, List, Any, Optional
 from bs4 import BeautifulSoup
-from dateutil.relativedelta import relativedelta
 import requests
-import logging
-import re
 from datetime import datetime
+from dateutil.parser import parse as dateutil_parse
+from dateutil.parser import ParserError
 from uk_bin_collection.uk_bin_collection.common import *
```

Ruff (0.15.20)

[error] `from uk_bin_collection.uk_bin_collection.common import *` used; unable to detect undefined names

(F403)